### ISSUE TITLE
Menu: first key press does not render focus styles

### ISSUE THREAD

**Issue by miroslavstastny** [microsoft/fluent-ui-react#344]
### Steps

1. Have a menu with three items,
2. click on the third item,
3. hit left arrow,
4. hit left arrow again.

### Expected Result
- Second item selected and highlighted after step 3.
- First item selected and highlighted after step 4.

### Actual Result
After the step 3, second item is correctly selected, but it is not highlighted 💣

![menu_keyboard](https://user-images.githubusercontent.com/9615899/46717794-6e5cad80-cc69-11e8-8651-e07ae9ab4a09.gif)

### Version
Current master, commit 14b860ec.

**Comment by sophieH29** [comment]
Closed as duplicate of https://github.com/stardust-ui/react/issues/434, since it is the more library-related issue than ```Menu```component itself.
